probers/script/probe.go

```diff
@@ -0,0 +1,23 @@
+package script
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+func (p *scriptconfig) probe() error {
+	if _, err := os.Stat(p.scriptfilepath); os.IsNotExist(err) {
+		p.healthy = false
+		p.exitError = fmt.Sprint(err)
+		return err
+	}
+	if err := exec.Command(p.scriptfilepath).Run(); err != nil {
```

Comment:
This doesn't actually run the command and it doesn't return an error type. Please check the package documentation :-)